Cargo.toml

```diff
@@ -1,3 +1,6 @@
+[package]
+license = "MIT"
```

Comment:
TIL — cargo toml's that define workspaces but not packages (as this one does) don't support the license key.

if you try to compile the project like this, it complains about 

```
error: failed to parse manifest at `/Users/rbalicki/code/isograph/Cargo.toml`

Caused by:
  missing field `name`
  in `package`
```

Can you revert this file change, and then this is good to go and we'll merge it!!!